cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -860,29 +818,16 @@ def _duplicate_block(
         )
 
         children_handled = False
+        # Allow an XBlock to do anything fancy it may need to when duplicated from another block.
+        load_services_for_studio(source_item.runtime, user)
+        children_handled = dest_block.studio_post_duplicate(
```

Comment:
This required every xblock to inherit from the StudioEditable mixin. (see envs/common.py)